src/lib.rs

```diff
@@ -40,9 +40,9 @@
 #[cfg(feature = "alloc")]
 extern crate alloc;
 #[cfg(feature = "alloc")]
-use alloc::{string::String, vec::Vec};
+use alloc::{string::String, vec, vec::Vec};
 
-use core::iter;
+use core::{iter, u8};
```

Comment:
I used u8 module because the u8::MAX associated constant requires a relatively new compiler.

See also: https://github.com/KokaKiwi/rust-hex/issues/55

Reply:
You have a point. Personally, I do not really care about MSRV (because I want to use the new features and updating the compiler is just a simple `rustup update`) so I did not think much about it.

https://github.com/KokaKiwi/rust-hex/pull/58#issuecomment-860209726